operator-common/src/test/java/io/strimzi/operator/common/model/LabelsTest.java

```diff
@@ -134,36 +134,6 @@ public void testWithUserLabels()   {
         assertThat(nonNullLabels.toMap(), is(expected));
     }
 
-    @Test
-    public void testWithUserLabelsFiltersKubernetesDomainLabelsWithExceptionPartOfLabel()   {
```

Comment:
It looks like you delete the old test but don't add any new one. Should we have some new test for the new feature? I know testing things with env. vars is not easy. But at least that the default continues to work by default as it was should be easy to test (and that is more or less what this test was doing I guess).

Reply:
there is another test which tests the same scenario with labels filtering but using the fromResource method and it covers the default exclusion pattern - https://github.com/strimzi/strimzi-kafka-operator/blob/main/operator-common/src/test/java/io/strimzi/operator/common/model/LabelsTest.java#L202